raddb/sites-available/abfab-tls

```diff
@@ -40,6 +40,43 @@ listen {
 	clients = radsec-abfab
 }
 
+# There needs to be a separated "listen" section for IPv6.
+# Typically it will be identical to the IPv4 one above, but there might be
+# some differences (e.g. if a different certificate or port is desired)
+listen {
+	ipaddr = ::
```

Comment:
it should be: `ipv6addr`

Reply:
Well, according to the documentation, ipaddr works (indeed, I've tested this configuration). A different thing is whether using `ipv6addr` is preferred. In that case I guess a similar case for `ipv4addr` is recommended